docs/pages/debugging/devtools-plugins.mdx

```diff
@@ -0,0 +1,201 @@
+
+  export default function App() {
+    const navigationRef = useNavigationContainerRef();
+
+    useReactNavigationDevTools(navigationRef);
+
+    return (
+      <NavigationContainer ref={navigationRef}>{/* ... */}</NavigationContainer>
+    );
+  }
+  ```
+  </Tab>
+</Tabs>
+
+Run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the React Navigation plugin. This will open the plugin's web interface, which will show your navigation history as you navigate through your app.
+
+### Apollo Client
+
+Inspired by [react-native-apollo-devtools](https://github.com/razorpay/react-native-apollo-devtools), Apollo Client dev tools plugin lets you example cache, queries and mutations for the Apollo Client.
+
+To use the plugin, first install the package:
+
+<Terminal cmd={['$ npx expo install @dev-plugins/apollo-client']} />
+
+Then pass your client instance to the plugin in your app's root component, or wherever you wrap the rest of your app in the `ApolloProvider`:
+
+```jsx App.js
+import { ApolloProvider, ApolloClient, InMemoryCache } from '@apollo/client';
+import { useApolloClientDevTools } from '@dev-plugins/apollo-client';
+
+const client = new ApolloClient({
+  uri: 'https://demo.test.com/',
+  cache: new InMemoryCache(),
+});
+
+export default function App() {
+  if (__DEV__) {
+    useApolloClientDevTools(client);
+  }
+
+  return (
+    <ApolloProvider>
+      {/* ... */}
+    </ApolloProvider>
+  )
+}
+```
+
+Run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the Apollo Client plugin. This will open the plugin's web interface, which will show your query history, cache, and mutations as your app performs Apollo Client operations.
+
+### React Query
+
+Inspired by [react-query-native-devtools](https://github.com/bgaleotti/react-query-native-devtools), the React Query Dev Tools Plugin lets you explore [TanStack Query](https://tanstack.com/query/latest/)'s data and queries, cache status, and refetch and remove queries from the cache.
+
+To use the plugin, first install the package:
+
+<Terminal cmd={['$ npx expo install @dev-plugins/react-query']} />
+
+Then pass your client instance to the plugin in your app's root component, or wherever you wrap the rest of your app in the `QueryClientProvider`:
+
+```jsx App.js
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { useReactQueryDevTools } from '@dev-plugins/react-query';
+
+const queryClient = new QueryClient({});
+
+export default function App() {
+  if (__DEV__) {
+    useReactQueryDevTools(queryClient);
+  }
+
+  return (
+    <QueryClientProvider>
+      {/* ... */}
+    </QueryClientProvider>
+  )
+}
+```
+
+Run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the React Query plugin. This will open the plugin's web interface, displaying queries as they are used in your app.
```

Comment:
```suggestion
In the terminal, run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the React Query plugin. This will open the plugin's web interface, displaying queries as they are used in your app.
```